manifests/mod/ssl.pp

```diff
@@ -10,33 +10,39 @@
   $ssl_random_seed_bytes   = '512',
   $ssl_sessioncachetimeout = '300',
   $apache_version          = $::apache::apache_version,
+  $ssl_mutex               = undef,
   $package_name            = undef,
 ) {
 
-  case $::osfamily {
-    'debian': {
-      if versioncmp($apache_version, '2.4') >= 0 {
-        $ssl_mutex = 'default'
-      } elsif $::operatingsystem == 'Ubuntu' and $::operatingsystemrelease == '10.04' {
-        $ssl_mutex = 'file:/var/run/apache2/ssl_mutex'
-      } else {
-        $ssl_mutex = "file:\${APACHE_RUN_DIR}/ssl_mutex"
+
+  if $ssl_mutex {
+    $ssl_mutex_real = $ssl_mutex
```

Comment:
why not just use [pick()](https://github.com/puppetlabs/puppetlabs-stdlib#pick) instead of this giant if/else?